### DISCUSSION TITLE
Parabolic street-mirror

### DISCUSSION THREAD

**Discussion by jumpjack** [ricktu288/ray-optics#155] [General]
Any idea about how to simulate how images reflect onto one of those parabolic mirrors placed at street crosses? I can't very understand how the option "seen by observer" works.
If I have an object passing in front of the mirror, can I simulate how it looks in the mirror?

**Comment by ricktu288** [comment]
"Seen by observer" just shows the images where an observe at that point can observer (assuming he/she can turn 360deg to observe to any direction). Note that an actual observer can know the distance to the object is by adjusting the focus of the eye/camera.

Since this is a 2D simulator, the actual image in this observer's retina is 1D, which can be approximately derived from where the images are for a given focus. If you really want the accurate representation of the irradiance map on the retina, you may try to make an eye manually by a lens and a detector, and turn on the irradiance map for the detector.
